packages/create-parser/tests/handlebars-utils.ts

```diff
@@ -20,5 +20,10 @@ test('compile template uses the `dependencyVersion` helper', async (t) => {
 });
 
 test('compile throws an error if template is not found', async (t) => {
-    await t.throws(handlebars.compileTemplate(path.join(__dirname, 'idontexist.hbs'), {}));
+    t.plan(1);
+    try {
```

Comment:
Why this change? `t.throws` doesn't work anymore?

Reply:
I'm fixing this, because this is not the case and it can be fixed using `t.throwsAsync`, but in some other cases, the problem is that the `reject` doesn't return an error, and `ava` now complains about that.